server/routes/pubDocument.tsx

```diff
@@ -235,3 +251,42 @@ app.get(
 		}
 	},
 );
+
+app.get(['/pub/:pubSlug/review/:historyKey/'], async (req, res, next) => {
+	if (!hostIsValid(req, 'community')) {
+		return next();
+	}
+	try {
```

Comment:
We have a `wrap` helper that should let us get rid of the try/catch here. But I see we haven't used it in a few other routes. Did you look into this already?